src/components/containers/dropdown.tsx

```diff
@@ -0,0 +1,46 @@
+import { FaChevronDown } from "react-icons/fa";
+import { Skeleton } from "../ui/skeleton";
+import clsx from "clsx";
+
+interface DropdownProps {
+  items: string[];
+  name: string;
+  isLoading?: boolean;
+  isAlwaysFullWidth?: boolean;
+}
+
+const Dropdown = ({
+  items,
+  name = "Item",
+  isLoading = false,
+  isAlwaysFullWidth = false,
+}: DropdownProps) => {
```

Comment:
to make it more readible, i prefer like this:
```ts
const Dropdown = (props: DropdownProps) => {
  // using object descruction
  const { items, name = "Item", isLoading = false, isAlwaysFullWidth = false } = props;
}
```

Reply:
also if possible, remove unnecessary default variable on that name/isLoading/isAlwaysFullWidth